src/diffusers/models/unet_motion_blocks.py

```diff
@@ -0,0 +1,1234 @@
+from typing import Any, Dict, Optional, Tuple
```

Comment:
Not too happy about creating a new file here. Most of the code should go either into the unet_3d_blocks or 2d_blocks code

Reply:
I think I can move it into the 2D blocks, since all the motion blocks are mods of the 2D ones.